composer.json

```diff
@@ -15,16 +15,16 @@
         {
             "name": "PrestaShop SA",
             "email": "contact@prestashop.com"
-        },{
-            "name": "Pablo Borowicz",
-            "email": "pablo.borowicz@prestashop.com"
         }
     ],
     "require": {
-        "php" : ">=5.4"
+        "php" : ">=7.2"
     },
     "require-dev": {
-        "phpunit/phpunit" : "4.*"
+        "phpunit/phpunit" : "8.*",
+        "codacy/coverage": "dev-master",
```

Comment:
Theoretically this exposes us to supply chain attacks.

Reply:
Don't know where this one comes from